bower.json

```diff
@@ -10,21 +10,21 @@
   ],
   "dependencies": {
     "purescript-prelude": "^4.0.0",
-    "purescript-web3": "^1.0.0",
-    "purescript-web3-generator": "^1.0.0",
+    "purescript-eth-core": "#upgrade",
+    "purescript-web3": "#upgrade",
+    "purescript-web3-generator": "#upgrade",
     "purescript-console": "^4.0.0",
     "purescript-errors": "^4.0.0",
     "purescript-node-process": "^6.0.0",
-    "purescript-debug": "^4.0.0",
-    "purescript-optparse": "f-o-a-m/purescript-optparse#v0.1.0",
+    "purescript-optparse": "^1.0.0",
```

Comment:
Would be great if you also bump optparse to `v2.0.0` (just published new version which fixed some time complexity issue and is faster now) (no code change would be needed)